plugins/common_event_format_logs.yaml

```diff
@@ -50,7 +50,6 @@ template: |
         - type: csv_parser
           parse_from: attributes.cef_headers
           header: 'version|device_vendor|device_product|device_version|signature_id|name|severity|extensions'
```

Comment:
I didn't bump the version of this plugin as the field I removed wasn't being used anyway so the way this plugin functioned didn't change.